app/src/main/java/com/xa/xpensauditor/ProfileActivity.java

```diff
@@ -30,7 +30,9 @@
 
 public class ProfileActivity extends AppCompatActivity {
 
-
+    /**
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.SummaryJavadocCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>First sentence of Javadoc is missing an ending period.